mac/macdeploy.sh

```diff
@@ -34,12 +40,22 @@ do
  install_name_tool -change $LIBRARY_NAME @executable_path/$LIBRARY_NAME $plugin
 done
 
-# prompt and optionally copy additional Qt native plugin(s) into bundle
-echo -n "Do you wish to redistribute known, missing, Qt Library plugins (y/n)? "
-old_stty_cfg=$(stty -g)
-stty raw -echo
-answer=$( while ! head -c 1 | grep -i '[yn]'; do true; done )
-stty $old_stty_cfg
+if [[ "$REDISTRIBUTE" == "QT" ]]; then
+  echo "$REDISTRIBUTE_QUESTION Yes"
+  answer="y"
+else
+   if [[ "$REDISTRIBUTE" == "NOQT" ]]; then
+     echo "$REDISTRIBUTE_QUESTION No"
+     answer="n"
+   else
+     # prompt and optionally copy additional Qt native plugin(s) into bundle
+     echo -n $REDISTRIBUTE_QUESTION
+     old_stty_cfg=$(stty -g)
+     stty raw -echo
+     answer=$( while ! head -c 1 | grep -i '[yn]'; do true; done )
```

Comment:
@Martii Maybe I didn't get your point correct. But,
1- 2nd parameter is optional no changes needed for other codes using `macdeploy.sh`

2- If you ask me, that question is useless and should be removed completely. Since `macdeploy.sh` is mainly used by developers and maintainers and I think they always want to bundle Qt libraries.